packages/nimble-components/src/table/types.ts

```diff
@@ -31,7 +31,11 @@ export type TableStringField<FieldName extends TableFieldName> = {
     [name in FieldName]: TableStringFieldValue;
 };
 
-export interface TableValidity {
+export interface ValidityObject {
```

Comment:
For naming consistency between table validity and column validity, could you add an empty interface called `TableColumnValidity` that extends `ValidityObject` and use that as the return type of `public get validity()` in `table-column/base/index.ts` and in `column-validator.ts`?

Reply:
Ok